### PR TITLE
Add Contribute index page for Indonesian localization

### PR DESCRIPTION
**Pull request by phanama**
This PR kickstarts the index page for #18336 
Requesting for proofreading.

/assign @irvifa 
/assign @girikuncoro

### CODE REVIEW

File: content/id/contribute/_index.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+content_template: templates/concept
+title: Berkontribusi ke dokumentasi Kubernetes
```
Comment (irvifa) [inline content/id/contribute/_index.md]:
```suggestion
title: Berkontribusi ke Dokumentasi Kubernetes
```

Comment (phanama) [reply content/id/contribute/_index.md]: Addressed here https://github.com/kubernetes/website/pull/18452/commits/8237a3870e695d3dd41d3f2411b5898008b21c5d

I also added an update to a mistranslation to Open issue -> I thought this was a noun, but it's actually a verb.
Thanks @wahyuoi for pointing this out!
https://github.com/kubernetes/website/pull/18452/commits/8237a3870e695d3dd41d3f2411b5898008b21c5d#diff-ffc999e20875f5664de07a3c269c7118R51

File: content/id/contribute/_index.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+---
```
Comment (sftim) [inline content/id/contribute/_index.md]: Did you mean: `content/id/docs/contribute/_index.md` ?

Comment (phanama) [reply content/id/contribute/_index.md]: Oh my, thanks for that! @sftim 
I've moved it to the right path.

File: content/id/docs/contribute/_index.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  mempublikasi konten atas nama organisasi Kubernetes.
+  Para _approver_ juga dapat mewakili SIG Docs dalam komunitas Kubernetes
+  yang lebih besar. Beberapa tugas seorang SIG Docs _approver_, seperti
+  mengkoordinasi sebuah perilisan, membutuhkan komitmen waktu yang signifikan.
+
+## Cara-cara untuk berkontribusi ke dokumentasi
+
+Daftar ini dibagi menjadi hal-hal yang dapat dilakukan oleh siapapun, hal-hal yang
+dapat dilakukan oleh anggota-anggota organisasi Kubernetes, dan hal-hal yang
+memerlukan tingat akses yang lebih tinggi serta pengetahuan terhadap proses-proses
```
Comment (girikuncoro) [inline content/id/docs/contribute/_index.md]: typo
```suggestion
memerlukan tingkat akses yang lebih tinggi serta pengetahuan terhadap proses-proses
```

**Comment by girikuncoro** [comment]
Everything looks good, just need to fix 1 minor typo and you are good to go. Thanks a lot for doing this, and welcome back from holiday!

File: content/id/docs/contribute/_index.md (hunk 3 above)
Comment (phanama) [reply content/id/docs/contribute/_index.md]: Fixed in https://github.com/kubernetes/website/pull/18452/commits/7ca1ba41e206ce53601506378b5c886d22ed3905

Thanks!

**Comment by girikuncoro** [comment]
Thanks a lot!

/lgtm
/approve

**Pull request merged**